gcp/README.md

```diff
@@ -42,8 +51,14 @@ An environment needs some resources created in the organization before the follo
    * The `gpii-infra` clone and the `exekube` clone should be siblings in the same directory (there are some references to `../exekube`).
 1. By default you'll use the RtF Organization and Billing Account.
    * You can use a different Organization or Billing Account, e.g. from a GCP Free Trial Account, with `export ORGANIZATION_ID=111111111111` and/or `export BILLING_ID=222222-222222-222222`.
+1. In the case of using a **dev** environment, be sure that the environment variable `$USER` is set to the same name used to name your dev project at GCP. In case of doubt ask to the ops team.
 1. `cd gpii-infra/gcp/live/dev`
 1. `rake`
+1. If it's the first time that you deploy the infrastructure you will be prompted to verify your identity at Google and allow permissions to your applications to perform modifications in your personal project at GCP. Go to the url shown and copy and paste the token once the application is authorized.
+1. Once finished all the GPII endpoints should be available at https://<service>.<your cluster name>.dev.gcp.gpii.net/
```

Comment:
I would add an example here to illustrate: "should be available at https://XXX, e.g. https://preferences.amatas.dev.gcp.gpii.net/".